### DISCUSSION TITLE
Feature request - Autocomplete with enum list

### DISCUSSION THREAD

**Discussion by Neogeekmo** [josdejong/svelte-jsoneditor#496] [Ideas]
Currently, the data passed in enum only allows for a dropdown list and doesn't provide the option to edit the values.

It would be necessary to have a mix between adding suggestions and making modifications.
![image](https://github.com/user-attachments/assets/bfcce025-93a5-463a-9b5a-8d501e508af9)

**Comment by josdejong** [comment]
Thanks for your suggestion, that makes sense. I'll move this as an "idea" into the "Discussions" section. Help implementing this would be welcome.

**Reply by Neogeekmo** [reply]
So I'll try to implement my own idea xD

**Comment by AlexRMU** [comment]
I think this is part of the https://github.com/josdejong/svelte-jsoneditor/discussions/366

**Reply by josdejong** [reply]
They are indeed related, though #366 is about autocompletion in `text` mode, and this idea #496 is about autocompletion in `tree` and `table` mode.
